android/filament-android/src/main/java/com/google/android/filament/Material.java

```diff
@@ -29,45 +29,101 @@
 import java.util.List;
 import java.util.Set;
 
+/**
+ * A Filament Material defines the visual appearance of an object. Materials function as a
+ * templates from which {@link MaterialInstance}s can be spawned. Use {@link Builder} to construct
+ * a Material object.
+ */
 public class Material {
     private long mNativeObject;
     private final MaterialInstance mDefaultInstance;
 
     private Set<VertexBuffer.VertexAttribute> mRequiredAttributes;
 
+    /** Supported shading models */
     public enum Shading {
+        /** No lighting applied, emissive possible */
```

Comment:
Maybe add links to the materials doc. For instance `https://google.github.io/filament/Materials.html#materialmodels/litmodel` for lit.